testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -204,9 +220,18 @@ def test_set_residues(self):
 
 
 class TestResnames(TestResidueAttr):
-    values = np.array(['ARG', 'LYS', 'VAL', 'POPG'], dtype=np.object)
+    values = np.array(['VAL', 'LYS', 'VAL', 'POPG'], dtype=np.object)
     attrclass = tpattrs.Resnames
 
+    def test_get_named_residue(self):
```

Comment:
This one is curious. The attr has a list of residue names but I'm unable to use the `_get_named_residue` function because it's accessing a `residue` attribute of the class that doesn't exist.